### PR TITLE
Implementando modal para la confirmacion de eliminacion de actividad

### PR DESCRIPTION
**Pull request by MarioRam0s**
## Description
Se implemento el diseño del modal de confirmación de eliminación de la actividad al igual que se hicieron cambios en la interfaz del menu y el select del menu
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (no functional changes, no api changes)
- [ ] This change requires a documentation update
- [ ] Other (please describe): 

## How Has This Been Tested?

y
yy

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in areas hard to understand.
- [ ] I have made the corresponding changes to the documentation.
- [x] My changes do not generate new warnings.
- [x] I have added tests that prove my solution is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] All dependent changes have been merged and published in subsequent modules.
- [x] I have reviewed my code and corrected any spelling mistakes.

### CODE REVIEW

File: lib/presentation/utils/constants_menu.dart [hunk 1]
```diff
@@ -1,85 +1,117 @@
 import 'package:flutter/material.dart';
 import 'package:hope_app/generated/l10n.dart';
 import 'package:hope_app/presentation/interfaces/interface.dart';
+import 'package:hope_app/presentation/utils/constants_desing.dart';
 
 final appMenuItemsDrawer = <MenuItem>[
```
Comment (SamBarFu) [inline lib/presentation/utils/constants_menu.dart:6]: Porque esta declaración de esta varaible esta diferente al resto?

más abajo tenes lo siguiente

final List<MenuItem> menuPacientTherapist = ...

Cuál es la diferencia de usar una o la otra?

**Review by SamBarFu — CHANGES_REQUESTED** [review]

File: lib/presentation/utils/constants_menu.dart (hunk 1 above)
Comment (LordMario) [reply lib/presentation/utils/constants_menu.dart:6]: Si no comprendo mal la observación , siendo mas claro, no estoy indicando el tipo de datos como en los demás?
Si es eso , se corregirá en el próximo PR

File: lib/generated/intl/messages_en.dart [hunk 2]
```diff
@@ -117,6 +119,9 @@ class MessageLookup extends MessageLookupByLibrary {
             MessageLookupByLibrary.simpleMessage(
                 "¿Está seguro de actualizar los datos?"),
         "Esta_seguro_de_avanzar_de_fase_a": m0,
+        "Esta_seguro_de_eliminar_permanentemente_la_actividad":
+            MessageLookupByLibrary.simpleMessage(
+                "¿Esta seguro de eliminar permanentemente la actividad?"),
```
[the remark this answers is not in the document]
Comment (LordMario) [reply lib/generated/intl/messages_en.dart:124]: Se corregira en el proximo PR

File: lib/generated/intl/messages_es_NI.dart [hunk 3]
```diff
@@ -117,6 +119,9 @@ class MessageLookup extends MessageLookupByLibrary {
             MessageLookupByLibrary.simpleMessage(
                 "¿Está seguro de actualizar los datos?"),
         "Esta_seguro_de_avanzar_de_fase_a": m0,
+        "Esta_seguro_de_eliminar_permanentemente_la_actividad":
+            MessageLookupByLibrary.simpleMessage(
+                "¿Esta seguro de eliminar permanentemente la actividad?"),
```
[the remark this answers is not in the document]
Comment (LordMario) [reply lib/generated/intl/messages_es_NI.dart:124]: Se corregira en el proximo PR

**Review by SamBarFu — APPROVED** [review]

**Pull request merged**